lua/entities/sent_deployableballoons.lua

```diff
@@ -34,13 +34,13 @@ CreateConVar('sbox_maxwire_deployers', 2)
 local DmgFilter
 
 local function CreateDamageFilter()
-	if DmgFilter then return end
+	if IsValid(DmgFilter) then return end
 	local DmgFilter = ents.Create("filter_activator_name")
```

Comment:
It shouldn't even be needed anymore. `InitPostEntity` should only fire once, where `Initialize` will fire every time the gamemode is reloaded (e.g. autorefresh).